static/templates/settings/notification_settings.hbs

```diff
@@ -2,41 +2,62 @@
     <form class="notification-settings-form">
         <div class="notification-reminder tip">{{#tr this }}Desktop notifications are triggered for messages that are offscreen when they arrive. Mobile and email notifications are triggered once you have been away from Zulip for a few minutes.{{/tr}}</div>
 
-        <div id="stream-notify-settings" class="subsection-parent">
-            <h3 class="inline-block">{{t "Stream messages" }}</h3>
-            <div class="alert-notification" id="stream-notify-settings-status"></div>
-            <p>{{t "Unless I say otherwise for a particular stream, I want:" }}</p>
+        <div id="general_notifications" class="subsection-parent">
+            <h3 class="inline-block">{{t "General notifications settings" }}</h3>
+            <div class="alert-notification" id="general-notify-settings-status"></div>
+            <table id="notification-table" class="table table-condensed table-bordered wrapped-table">
+                <thead>
+                    <tr>
+                        <th rowspan="2" width="40%"></th>
+                        <th colspan="2" width="30%">{{t "Desktop"}}</th>
+                        <th rowspan="2" width="15%" class="{{#if show_push_notifications_tooltip}}control-label-disabled{{/if}}">
+                            {{t "Mobile"}}
+                            {{#if (not page_params.pm_mention_notification_settings) }}
+                            <i class="fa fa-question-circle settings-info-icon" data-toggle="tooltip"
+                              title="{{t 'Mobile push notifications are not configured on this server.' }}"></i>
+                            {{/if}}
+                        </th>
+                        <th rowspan="2" width="15%">{{t "Email"}}</th>
```

Comment:
I'd like to have `wildcard_mentions_notify` be a column here; once you look at @ryanreh99's PR, you'll see why (there's a per-stream setting we want to have in a similarly styled table)!  We can use `@all` as a label if we need something small there.

We just need to configure this widget to have a blank space or disabled checkbox or something for the "Private messages" case of it, since there's nothing for that to do.

Reply:
Current UI is created while being in the scope of https://github.com/zulip/zulip/issues/12182, and I also expected to have a `wildcard_mentions_notify` column here, I thought to introduce it in future as the mockup Shubham shared with me has individual streams too, but looks like that part is already done by @ryanreh99 which is great.